spec/presenter_spec.rb

```diff
@@ -0,0 +1,62 @@
+require './lib/presenter'
+
+describe 'presenter' do
+  class DisplayLogger
+    attr_reader :log
+
+    def initialize
+      @log = []
+    end
+
+    def display(message)
+      log << message
+    end
+  end
+
+  let(:displayer) { DisplayLogger.new }
```

Comment:
I personally prefer to just use doubles instead of mocks for these type of tests

Reply:
It's lighter weight and uses rspec

Reply:
One of my restrictions is to not use test doubles

Reply:
To clarify, I did add a constraint that nothing from the `rpec-mocks` package could be used.

Reply:
👍 Sorry, was unaware!